openpype/modules/deadline/deadline_module.py

```diff
@@ -10,7 +10,15 @@ def initialize(self, modules_settings):
         # This module is always enabled
         deadline_settings = modules_settings[self.name]
         self.enabled = deadline_settings["enabled"]
-        self.deadline_url = deadline_settings["DEADLINE_REST_URL"]
+        deadline_url = deadline_settings.get("DEADLINE_REST_URL")
```

Comment:
I think that module should have attribute which holds all defined deadline URLs. Not only default.